interface/src/app/analyze/analyze.js

```diff
@@ -52,6 +53,21 @@ angular.module('adage.analyze', [
         return classList;
       },
 
+      item_tooltip: function(search_item) {
```

Comment:
Ok, sounds good, if this will all happen with with the refactoring, and it is being tracked.